spot-client/src/spot-tv/ui/components/exit-electron/ExitVerification.test.js

```diff
@@ -0,0 +1,130 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+
+import { mockT } from 'common/test-mocks';
+
+jest.mock('spot-tv/app-state');
+import { fetchExitPassword } from 'spot-tv/app-state';
+
+import { ExitVerification } from './ExitVerification';
+
+
+const SELECT_EXIT_CONFIRMATION = { 'data-qa-id': 'exit-electron-confirmation' };
+const SELECT_LOADING_SCREEN = { 'data-qa-id': 'exit-electron-loading' };
+const SELECT_PASSWORD_PROMPT = { 'data-qa-id': 'exit-electron-password-prompt' };
+
+const SELECT_PASSWORD_INPUT = { 'data-qa-id': 'exit-electron-password-input' };
+
+const SELECT_CANCEL_BUTTON = { 'data-qa-id': 'exit-electron-cancel' };
+const SELECT_CONFIRM_BUTTON = { 'data-qa-id': 'exit-electron-confirm' };
+
+describe('ExitVerification', () => {
+    let exitComponent, onCancel, onPasswordInvalid, onVerification, store;
+
+    beforeEach(() => {
+        store = createStore((state = { config: {} }) => state);
+        onCancel = jest.fn();
+        onPasswordInvalid = jest.fn();
+        onVerification = jest.fn();
+    });
+
+    const mountExitComponent = () => {
+        exitComponent = mount(
+            <Provider store = { store }>
```

Comment:
I think don't the store is needed. I think if you use the non-connected ExitVerification then all the values can be passed in manually. Unless that is your wanting to test the full integration with redux.

Reply:
Right, I will try without the store.  It's using disconnected version anyway.

Reply:
Turns out it is need for some of the nested components: `Background`

Reply:
What if you did a shallow mount?

Reply:
Nevermind, I think that might screw up the buttons.